### ISSUE TITLE
LibGuides: Link alt text

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#50]
[Siteimprove link](https://my2.siteimprove.com/Accessibility/624596/77184128804/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r11&ruleId=11&issueKind=1&tagFilterType=2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

[csv](https://github.com/user-attachments/files/18713385/SC.-.Library-Pages.with.a.specific.issue-Table.4.csv)

I think most or all of these issues results from `<a>` tags that aren't properly closed. Most browsers resolve this issue for our users, but the siteimprove crawler returns this issue (it thinks subsequent elements are also links and can't find their link text).

That being said, `<a>` tags should be properly formed so we have valid HTML that is usable by all assistive technology, so those should probably be fixed as an A11y issue.

**Comment by JessWaggs** [comment]
Book from the Catalog content items in LibGuides are missing both the alt text for the cover art image, which is an active link. I looked at the edit options Springshare provides for Book from the Catalog items and I did not see a way to enter alt text for the cover art images.

I've reported the issue to Springshare on 7/08/25.
